geode-protobuf/src/main/java/org/apache/geode/protocol/protobuf/operations/RemoveRequestOperationHandler.java

```diff
@@ -33,7 +33,7 @@
 
   @Override
   public ClientProtocol.Response process(SerializationService serializationService,
-      ClientProtocol.Request request, Cache cache) {
+                                         ClientProtocol.Request request, Cache cache) {
```

Comment:
Does this pass spotless?  I thought all of these wonky indentations were actually introduced by spotlessApply?